src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -254,6 +254,7 @@ struct SvgStyleGradient
     SvgLinearGradient* linear;
     Matrix* transform;
     Array<Fill::ColorStop *> stops;
+    float prevOffset;
```

Comment:
@JSUYA I added this so I don't need to call the ```colorStops``` getter every time (we need to know the previous offset value before we add a new 'stop').